lib/fontcustom/scripts/generate.py

```diff
@@ -140,6 +141,6 @@ def createGlyph( name, source, code ):
 
 finally:
     manifestfile.seek(0)
-    manifestfile.write(json.dumps(manifest, indent=2, sort_keys=True))
+    manifestfile.write(json.dumps(manifest, indent=2, sort_keys=True).encode('utf-8').decode('utf-8'))
```

Comment:
Could you help me understand why you're using utf-8 here, but latin-1 everywhere else? I mostly want to make sure this was intentional.